jupyter/fuse/fuse_mount.py

```diff
@@ -45,6 +46,7 @@
             # "curldbg",
         ]
         + (["-o", "ro"] if bucket["mode"] == "RO" else [])
+        + (["-o", "use_path_request_style"] if aws_endpoit else [])
```

Comment:
Adding the option if we have an aws endpoint is not very understandable. 

Is the logic "if minio, then add the option" ? If it is the case, maybe we miss an environment variable that explicitly states that we are in minio mode.

If it is too complex, then at least let's add some comments to explain why we are doing this.